shared/src/starterKits.ts

```diff
@@ -0,0 +1,27 @@
+import { Industries } from "./industry";
```

Comment:
Dust, I have a question: Why is this living in a shared place if we only utilize it on the web?

Reply:
Yeah thats a good point, initially we had this script in API but moved it to web so now theres no need. Changed in my last push up